CHANGELOG.md

```diff
@@ -13,6 +13,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+## [1.22.3]
+
+### Added
+
+- Added dart generation to Visual Studio Code extension.
```

Comment:
There shouldn't be new features in a patch release. Could you update the changelog? Also, make sure this feature isn't in the patch